classes/helper/login_helper.php

```diff
@@ -531,4 +532,18 @@ public function clear_username_in_auth_global_vars() {
         $frm = null;
         set_moodle_cookie('');
     }
+
+    /**
+     * Get user failed login count
+     *
+     * @param $userName
+     * @return int|null
+     * @throws \coding_exception
+     * @throws \dml_exception
+     */
+    public function get_user_login_failCount ($userName){
```

Comment:
Please can you fix spacing on this line - there should be no space before the opening bracket, and the opening curly brace should be on the next line (or at least have a space before). This is part of PSR-2